README.md

```diff
@@ -359,7 +295,7 @@ s.Close()
 
 ## Custom made components
 
-Parts of this package may not fulfill your application need, either it can be that the event or snapshot stores uses the wrong database for storage.
+Parts of this package may not fulfill your application need, either it can be that the event store uses the wrong database for storage.
```

Comment:
Can a database be wrong, or maybe there doesn't exists an adapter for the database that you use?
```suggestion
Parts of this package may not fulfill your application need, for instance an event store adapter for your database of choice.
```